tests/test_load_model.py

```diff
@@ -14,39 +12,6 @@
 )
 
 
-def test_cyclic_diagram_bug(element_factory, modeling_language, test_models):
-    """Load file.
-
-    This does not nearly resemble the error, since the model should be
-    loaded from within the mainloop (which will delay all updates).
-    """
-    path = test_models / "dbus.gaphor"
-    with path.open() as file_obj:
-        load(file_obj, element_factory, modeling_language)
-
-
-def test_cyclic_diagram_bug_idle(element_factory, modeling_language, test_models):
-    """Load file in gtk main loop.
-
-    This does not nearly resemble the error, since the model should be
-    loaded from within the mainloop (which will delay all updates).
-    """
-
-    def handler():
-        try:
-            path = test_models / "dbus.gaphor"
-            with path.open() as file_obj:
-                load(file_obj, element_factory, modeling_language)
-        finally:
-            Gtk.main_quit()
-
-    assert GLib.timeout_add(1, handler) > 0
-
-    ctx = GLib.main_context_default()
-    while ctx.pending():
-        ctx.iteration(False)
-
-
 def test_association_ends_are_set(element_factory, modeling_language, test_models):
```

Comment:
It looks like tests are failing due to the element_factory fixture being missing. If I import it, it is showing as an unused import though.